geocoding/generate_pincode.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import math
+import pathlib
+
+import fire
+import googlemaps
+import pandas as pd
+from tqdm import tqdm
+
+logging.basicConfig(
+    filename="pincoding.log",
+    filemode="a",
+    format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
+    datefmt="%H:%M:%S",
+    level=logging.INFO,
+)
+
+logging.info("Logger Setup Complete")
+
+
+def send_reverese_geocoding_request(lat, lng, gmaps):
+    """Responsible for sending the request to GMaps API to reverser geocode
+    
+    Args:
+        lat (float): Latitude
+        lng (float): Longitude
+        gmaps (Object): gmaps object with declared API key
+    
+    Returns:
+        dict: Response recieved from API
+    """
+    successful_api_hit = True
```

Comment:
`successful_api_hit` can be more precise? Is this 200 OK? Or that you got valid data back? It'd help if the variable name distinguished it.

Reply:
Replaced the name `successful_api_hit` with `valid_data_received`